src/views/ZaaktypeEditView.tsx

```diff
@@ -0,0 +1,178 @@
+        enqueueSnackbar('Informatieobjecttypen zijn succesvol gekoppeld', {
+          variant: 'success',
+          preventDuplicate: true,
+          autoHideDuration: 1000,
+          anchorOrigin: { horizontal: 'right', vertical: 'top' },
+        });
+        setNewRelations(relations);
+      }
+      if (data.status === 'failed') {
+        enqueueSnackbar('Een aantal velden zijn niet gewijzigd', {
+          variant: 'error',
+          hint: 'Probeer het nogmaals',
+          preventDuplicate: true,
+          anchorOrigin: { horizontal: 'right', vertical: 'top' },
+        });
+      }
+    } catch (_error) {
+      // Add error handling
+    }
+  };
+
+  const handleQuit = () => {
+    const sameData = true;
+
+    const navigateQuit = () => {
+      closeSnackbar();
+      navigate('..', { relative: 'path' });
+    };
+
+    if (!sameData) {
+      enqueueSnackbar('Er zijn niet opgeslagen wijzigingen', {
+        preventDuplicate: true,
+        variant: 'unsavedChanges',
+        hint: 'Weet je zeker dat je wilt annuleren?',
+        persist: true,
+        action: () => (
+          <Box sx={{ p: 2 }}>
+            <Typography gutterBottom variant="body2">
+              Zaaktype toch opslaan?
+            </Typography>
+            <Button size="small" color="primary" startIcon={<Close />} onClick={navigateQuit}>
+              Ja, ik wil annuleren
+            </Button>
+          </Box>
+        ),
+        anchorOrigin: {
+          vertical: 'top',
+          horizontal: 'right',
+        },
+      });
+    } else {
+      navigateQuit();
+    }
+  };
+
+  return (
+    <Stack direction={'row'} flexWrap={'wrap'} width={'100%'} spacing={spacings.xlarge} useFlexGap>
+      <Stack
+        direction={'row'}
+        justifyContent={'space-between'}
+        alignItems={'center'}
+        flexWrap={'wrap'}
+        width={'100%'}
+        spacing={spacings.large}
+        useFlexGap
+      >
+        <Typography variant="h1">
+          {loadingState ? <Skeleton variant="text" width={200} /> : value?.zaaktype?.omschrijving}
+        </Typography>
+        <Stack direction={'row'} spacing={spacings.small}>
+          <Button variant="outlined" onClick={handleQuit}>
+            Annuleren
+          </Button>
+          <Button variant="contained" onClick={handleSave} disableElevation>
+            Opslaan
+          </Button>
+        </Stack>
+        <Stack spacing={spacings.small} width="100%">
+          <Typography variant="h2">
+            {loadingState ? (
+              <Skeleton variant="text" width={320} />
+            ) : (
+              'Koppel informatieobjecttypen'
+            )}
+          </Typography>
+          <Typography variant="body2">
+            {loadingState ? (
+              <Skeleton variant="text" width={340} />
+            ) : (
+              'Hier kunt u de gekoppelde informatieobjecttypen wijzigen.'
+            )}
+          </Typography>
+          <BulkEditor
```

Comment:
This would be a nice place to pass in a generic format. Somewhere else in this file we can create it based on our domain specific context.

Reply:
✅ => 🔥 BulkEditor and add domain specific logic to `ZaaktypenEditView`